changelog/manu-subscribeToSubnets.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- `subscribeToSubnets`: Remove unused parameter.
```

Comment:
Suggest using `Ignored` instead of `Fixed` here.

Reply:
Fixed in 9290bc7b144d3246e5c67a374164ae13ceef53b8.